pycstruct/pycstruct.py

```diff
@@ -460,6 +476,42 @@ def create_empty_data(self):
     buffer = bytearray(self.size())
     return self.deserialize(buffer)
 
+  def dtype(self):
```

Comment:
Could you add some comments for this method. I would like the comment to include:

- What it does (i.e. used for serialization/deserialization using numpy structured arrays)
- Why to use it (I guess performance is the main reason)
- Limitations (Nested structs, enums etc)
- Some examples on how to use it

You can check that the documentation looks ok by entering the doc directory and write "make html".